tools/scan-build-py/README.md

```diff
@@ -41,6 +41,31 @@ goes like this:
 Use `--help` to know more about the commands.
 
 
+How to use the experimental Cross Translation Unit analysis
+-----------------------------------------------------------
+
+To run the CTU analysis, a compilation database file has to be created:
+
+    $ intercept-build <your build command>
+
+To run the Clang static analyzer against a compilation database
+with CTU analysis enabled, execute:
+    
+    $ analyze-build --ctu
```

Comment:
Don't you need to pass the compilation database somehow? Maybe a sample invocation that could work without adding/removing parameters might be useful.

Reply:
It is indeed possible to pass the compilation database by using --cdb, but that is optional. The database that analyze-build tries to use by default is 'compile_commands.json', which is the default output of the intercept-build command (could be changed with --cdb as well). I was trying to keep this part as light as possible, and the simplest usage is really just calling 'intercept-build <build command> && analyze-build --ctu'. Introducing new options like --cdb would just make it more complicated in my opinion. Happy to update it with further details though.

Reply:
I see. In this case let's keep it minimal.